packages/fxa-auth-server/scripts/transfer-users/apple.js

```diff
@@ -124,8 +121,6 @@ export class AppleUser {
       // if they do, error out.
       const msg = `Uid not found: ${this.uid}`;
       console.error(msg);
-      this.setFailure(err);
-      return;
```

Comment:
We don't need to error out early here anymore?

Reply:
No we don't, these users were informed we were migrating account so still need to create it. I'll update comment.